src/about/settings/index.js

```diff
@@ -28,7 +55,99 @@
 
   // View
 
-  const viewSetting = (name, value) => html.div({
+  const viewField = props => {
+    return html.span({
+      key: 'field',
+      style: {
+        position: 'relative',
+        overflow: 'hidden'
+      }
+    },[
+      html.span({
+        key: 'sizer',
+        style: {
+          left: 0,
+          color: 'red',
+          whiteSpace: 'pre',
+          opacity: 0,
+        }
+      }, props.value),
```

Comment:
> Why do you need that?

I want input field to resize with the size of input, to achieve that input width is 100% and positioned absolute, while there is also a hidden span that has same style as input but isn't absolute there for causing it's parent to resize.